package.json

```diff
@@ -78,28 +78,28 @@
   "homepage": "https://github.com/Azure/avocado#readme",
   "dependencies": {
     "@azure/openapi-markdown": "^0.9.1",
-    "@ts-common/async-iterator": "^0.2.0",
+    "@ts-common/async-iterator": "^0.2.2",
     "@ts-common/commonmark-to-markdown": "^1.2.0",
     "@ts-common/fs": "^0.2.0",
-    "@ts-common/iterator": "^0.3.1",
-    "@ts-common/json": "^0.3.0",
-    "@ts-common/json-parser": "^0.7.0",
+    "@ts-common/iterator": "^0.3.6",
+    "@ts-common/json": "^0.3.1",
+    "@ts-common/json-parser": "^0.8.0",
     "@ts-common/string-map": "^0.3.0",
     "commonmark": "^0.29.0",
     "js-yaml": "^3.13.1",
     "node-object-hash": "^1.4.2"
   },
   "devDependencies": {
     "@types/js-yaml": "^3.12.1",
-    "@types/jest": "^24.0.12",
+    "@types/jest": "^24.0.15",
     "@types/node": "^10.14.6",
     "jest": "^24.8.0",
     "jest-junit": "^6.4.0",
-    "prettier": "^1.17.0",
-    "tslint": "^5.16.0",
+    "prettier": "^1.18.2",
+    "tslint": "^5.18.0",
     "tslint-config-prettier": "^1.18.0",
     "tslint-config-standard": "^8.0.1",
-    "tslint-immutable": "^5.5.2",
```

Comment:
Why do these dependences need to update? And 'tslint-immutable' package seems have a major version update. Whether this change will involve other problems or breaking changes?

Reply:
tslint-immutable is a tool so there will be no breaking changes. Also, we are trying to keep use the latest stable packages as much as we can.